### PR TITLE
Enable Scheduler feature-toggle by default

### LINKED ISSUE #322: Enable Scheduler feature-toggle by default
Depends on:

- [x] https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/pull/5

- [x] https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/pull/6

- [x] https://github.com/spring-cloud/spring-cloud-dataflow/issues/2404

- [x] https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/issues/247

- [x] https://github.com/spring-cloud/spring-cloud-dataflow/issues/2405

- [x] https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/issues/4

### PR DESCRIPTION
**Pull request by chrisjs**
Resolves #322

### CODE REVIEW

**Comment by chrisjs** [comment]
to test will require at least:

- [x] https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/pull/5

- [x] https://github.com/spring-cloud/spring-cloud-dataflow/issues/2404

**Comment by cppwfs** [comment]
Need to create a CI environment before we can merge this PR. https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/issues/4

File: src/kubernetes/server/server-deployment.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@ spec:
           value: 'false'
         - name: SPRING_CLOUD_DATAFLOW_FEATURES_ANALYTICS_ENABLED
           value: 'true'
+        - name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
```
Comment (cppwfs) [inline src/kubernetes/server/server-deployment.yaml:38]: I think we should have this enabled by default.   But that differs from CF and the upcoming local.  @sabbyanandan can we make an exception here?

**Comment by cppwfs** [comment]
EEK!  Ignore my link above for acceptance tests.   I should have read the detail.   What I meant to say is that we need to have a bamboo build for Spring Cloud Scheduler Kubernetes so its bits can be in the repo.  That is why the travis build below is failing.

**Comment by chrisjs** [comment]
artifacts are now being published & travis is happy again

**Comment by chrisjs** [comment]
merged in ff44a4

**Pull request closed without merging**